### PR TITLE
[fix] Update styling for RadioButton and LeftAlignContainer components

### PR DESCRIPTION
**Pull request by selene-huang**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
- Updated incorrect flex property on Filter Modal in the transactions flow
- Updated Radio Button styling in the Sort Modal to accommodate long strings

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"
image
image

## How to review
[//]: # "Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"
Run `npx expo start` and navigate to the invoices flow. Open the Filter modal and check that all components are correctly styled. Open the Sort modal and check that all components are correctly styled.

### Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

Fixes a styling error introduced in #95 and improves Radio Button styling from #54 

[//]: # "This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation."
🥦 CC: @sauhardjain

### CODE REVIEW

File: src/screens/transactions/TransactionsScreen.tsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ export default function TransactionsScreen({
     'Amount: Low to High',
     'Date: Newest',
     'Date: Oldest',
+    'eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee',
```
Comment (sauhardjain) [inline src/screens/transactions/TransactionsScreen.tsx]: can we get rid of this lol

**Review by sauhardjain — APPROVED** [review]
